tests/HashidsTest.php

```diff
@@ -327,9 +327,21 @@ public function testBigNumberDecode($number, $hash)
      */
     public function testBehaviourForDifferentBCMathAccuracy()
     {
-        bcscale(2);
         $hashids = new Hashids('this is my salt', 12);
         $encoded = $hashids->encode(1);
         $this->assertEquals('DngB0NV05ev1', $encoded);
     }
+    
+
+    public function testSameResult()
+    {
+        bcscale(2);
+        $hashids = new Hashids('megas', 30);
+        $data[0] = $hashids->encode(215);
```

Comment:
@galamz you need to add `$this->assertSame($data[0], $data[1])` to tell PHPUnit to test the issue.

Reply:
thank you